neo/Consensus/ConsensusService.cs

```diff
@@ -124,7 +124,7 @@ private void CheckExpectedView(byte viewNumber)
                     // Communicate the network about my agreement to move to `viewNumber`
                     // if my last change view payload, `message`, has NewViewNumber lower than current view to change
                     if (message is null || message.NewViewNumber < viewNumber)
-                        localNode.Tell(new LocalNode.SendDirectly { Inventory = context.MakeChangeView() });
+                        localNode.Tell(new LocalNode.SendDirectly { Inventory = context.MakeChangeView(ChangeViewReason.WrongExpectedView) });
```

Comment:
I think it sends `ChangeView` because everyone wants to change view. There is nothing wrong.

Reply:
Hi @erikzhang Yoda master,

I think that if the reason is something like invalid tx there will be no reason for recovery, for example.

Reply:
`ConsensusChange`?

Reply:
`Follow`?

Reply:
@erikzhang, my comment above was wrong...

I just remembered now that we already created a special payload `RecoveryRequest`.
In this sense, I am not sure about a real use of this Reason.

Should we revert, @shargon? Or do you have an idea for using it?
@igormcoelho